user-manual/import-export/upload-digital-object.rst

```diff
@@ -191,6 +193,186 @@ your :term:`digital object`.
 
 :ref:`Back to top <upload-digital-object>`
 
+
+.. _digital-object-metadata:
+
+Digital object metadata
+=======================
+
+When you link a digital object to an archival description, the "Digital
+object metadata" section on the archival description view page will include
+an "Access Copies" subsection.
+
+   .. NOTE::
+
+      An unauthenticated user will not see a digital object and its "Access
+      copies" metadata if the archival description it is attached to has a
+      "draft" instead of "published" status.
+
+1. Clicking on the "Access copies" subsection label link will collapse or 
+   expand the subsection. 
+
+2. The "Access Copies" subsection will include metadata about the "Master
+   file", "Reference copy", and "Thumbnail copy" representations of the
+   digital object. Specifically: filename, media type, mime-type, filesize,
+   and uploaded (the timestamp for when the digital object was uploaded to
+   AtoM).
+
+   .. image:: images/access-copies-section.*
+      :align: center
+      :width: 90%
+      :alt: digital object metadata with access enabled for all representations
+
+3. An administrator user can toggle the display of specific "Digital object 
+   metadata" section fields on or off in the **Admin > Visible elements >
+   Digital object metadata area** section. This only affects display for 
+   unauthenticated users. Any authenticated users will see all metadata elements in each section as long as those fields contain a value.
```

Comment:
Might be good to add a link to the Visible elements documentation here? I.e., 

```
For more information, see: :ref:`visible-elements`. 
```

Also missing a wrap on the last line :)

Reply:
✓